fs/base.py

```diff
@@ -1244,7 +1244,7 @@ def setbytes(self, path, contents):
         with closing(self.open(path, mode="wb")) as write_file:
             write_file.write(contents)
 
-    def setbinfile(self, path, file):
+    def upload(self, path, file):
```

Comment:
Like `download`, I think `upload` better describes what this does than the clumsy `setbinfile`.